sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/JsonSupport.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk
+
+import com.google.protobuf.any.{ Any => ScalaPbAny }
+import com.akkaserverless.javasdk.{ JsonSupport => JavaJsonSupport }
+
+import scala.jdk.OptionConverters._
+import scala.reflect.ClassTag
+
+object JsonSupport {
```

Comment:
We should also register the `com.fasterxml.jackson.module.scala.DefaultScalaModule`. Easiest is probably to just add that dependency and register it in the `javasdk.JsonSupport.objectMapper`.

Reply:
Alternatively, we could be a bit more sophisticated and only have that dependency in the scalasdk and register it from here.

Reply:
Would `scala.DefaultScalaModule` mean a new dependency jar for the javasdk package? If so, better to avoid  it and only add to the Scala SDK project.

Reply:
Now setting it from the Scala JsonSupport object constructor